src/Block/Filter.hs

```diff
@@ -15,6 +16,9 @@ data DatumFilter
   | AddressFilter Text
   deriving stock (Show)
 
+instance Default DatumFilter where
```

Comment:
My personal opinion: I'm not a big fan of `data-default`. I'd personally go with `defaultDatumFilter`, but I'll leave the decision to you